packages/ibm-products/src/components/CoachmarkFixed/CoachmarkFixed.js

```diff
@@ -0,0 +1,209 @@
+ */
+
+// Import portions of React that are needed.
+import React, { useEffect, useRef, useState } from 'react';
+
+// Other standard imports.
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { createPortal } from 'react-dom';
+import { CoachmarkOverlay } from '../Coachmark/CoachmarkOverlay';
+import { CoachmarkTagline } from '../Coachmark/CoachmarkTagline';
+import { CoachmarkContext } from '../Coachmark/utils/context';
+import { COACHMARK_OVERLAY_KIND } from '../Coachmark/utils/enums';
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg /*, carbon */ } from '../../settings';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--coachmark-fixed`;
+const componentName = 'CoachmarkFixed';
+
+const defaults = {
+  onClose: () => {},
+  theme: 'light',
+  tagline: '',
+};
+
+/**
+ * Fixed coachmarks are used to call out specific functionality or concepts
+ * within the UI that may not be intuitive but are important for the
+ * user to gain understanding of the product's main value and discover new use cases.
+ * This variant allows the a coachmark overlay to be displayed by interacting with the tagline.
+ */
+export let CoachmarkFixed = React.forwardRef(
+  (
+    {
+      // The component props, in alphabetical order (for consistency).
+
+      children,
+      className,
+      onClose = defaults.onClose,
+      portalSelector,
+      tagline = defaults.tagline,
+      theme = defaults.theme,
+      // Collect any other property values passed in.
+      ...rest
+    },
+    ref
+  ) => {
+    const overlayRef = useRef();
+    const portalNode = portalSelector
+      ? document.querySelector(portalSelector) ?? document.querySelector('body')
+      : document.querySelector('body');
+    const [isOpen, setIsOpen] = useState(false);
+    const [shouldResetPosition, setShouldResetPosition] = useState(false);
+    const [targetRect, setTargetRect] = useState();
+    const [targetOffset, setTargetOffset] = useState({ x: 0, y: 0 });
+    const [fixedIsVisible, setFixedIsVisible] = useState(false);
+
+    const handleClose = () => {
+      setFixedIsVisible(false);
+    };
+
+    const handleTransitionEnd = (e) => {
+      if (e.propertyName === 'transform' && !fixedIsVisible) {
+        setIsOpen(false);
+        onClose();
+      }
+    };
+
+    const handleTargetClick = (e) => {
+      setTargetRect(e.target.getBoundingClientRect());
+      setTargetOffset({ x: e.target.offsetLeft, y: e.target.offsetTop });
+      // reset position for all other kinds
+      setIsOpen(false);
+      setShouldResetPosition(true);
+    };
+
+    const contextValue = {
+      buttonProps: {
+        'aria-expanded': isOpen,
+        onClick: handleTargetClick,
+        // Compensate for accidental open/close on double-click.
+        // Only open on double-click.
+        onDoubleClick: handleTargetClick,
+      },
+      closebuttonProps: {
```

Comment:
Haha! I think this is the culprit. So I think if we change `closebuttonProps` to `closeButtonProps`, we should be able to remove this from cspell. I think there are 5 instances (`Coachmark.js`, `CoachmarkFixed.js`, `CoachmarkOverlayElements.js`, and `CoachmarkStack.js`).

```suggestion
      closeButtonProps: {
```